sound/soc/sof/ipc4-topology.c

```diff
@@ -2413,9 +2415,15 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PPL_ID_MASK;
 		msg->extension |= SOF_IPC4_MOD_EXT_PPL_ID(pipe_widget->instance_id);
+
+		dev_dbg(sdev->dev, "Create widget %s ID %#x instance %d - pipe %d - core %d\n",
+			swidget->widget->name, module_id, swidget->instance_id,
+			swidget->pipeline_id, swidget->core);
+	} else {
+		dev_dbg(sdev->dev, "Create widget %s instance %d - pipe %d - core %d\n",
+			swidget->widget->name, swidget->instance_id, swidget->pipeline_id,
+			swidget->core);
```

Comment:
@btian1, can you at least tell why are you dismissing my comment and request?

Would not be beneficial to print this differently:
if ()
`Create widget for module: %s, ID: %d, instance: %d, pipeline: %d, core: %u`
else
`Create widget for pipeline: %s, instance: %d, pipeline: %d, core: %u`

Reply:
from my perspective, we already add two similar line logs(if from code optimize perspective it is not good, however, I can't name module_id if without separate print), I don't want to add two same C source code, current implementation is also fine and without logical error, right?

Reply:
What I'm asking for is readability and consistency, what about:
```c
	if (swidget->id != snd_soc_dapm_scheduler) {

		...

		dev_dbg(sdev->dev,
			"Create widget %s (ID %d, instance %d) - pipe %d - core %d\n",
			swidget->widget->name, module_id, swidget->instance_id,
			swidget->pipeline_id, swidget->core);
	} else {
		dev_dbg(sdev->dev,
			"Create widget %s (instance %d) - pipe %d - core %d\n",
			swidget->widget->name, swidget->instance_id,
			swidget->pipeline_id, swidget->core);
	}
```

Reply:
iow, it is not only a python code which is going to look at the logs..

Reply:
@ujfalusi @bardliao @aiChaoSONG , logs looks like below, please check whether meet you expecatation:
![image](https://github.com/thesofproject/linux/assets/17277136/74ac9ea6-b2f9-48bd-859d-1a7b77882553)

Reply:
> @ujfalusi @bardliao @aiChaoSONG , logs looks like below, please check whether meet you expecatation: ![image](https://user-images.githubusercontent.com/17277136/284162769-74ac9ea6-b2f9-48bd-859d-1a7b77882553.png)

The log looks good to me.

Reply:
Please add comma between the ID and instance prints.

But if I think about it, the print should be reflecting what is important (iow, sent to the firmware).
The `pipeline_id` is only used by the Linux driver, so it might be better to print something like this:
```
	if (swidget->id != snd_soc_dapm_scheduler) {

		...

		dev_dbg(sdev->dev,
			"Create widget %s (pipe %d) - ID %d, instance %d, core %d\n",
			swidget->widget->name, swidget->pipeline_id, module_id,
			swidget->instance_id, swidget->core);
	} else {
		dev_dbg(sdev->dev,
			"Create widget %s (pipe %d) - instance %d, core %d\n",
			swidget->widget->name, swidget->pipeline_id,
			swidget->instance_id, swidget->core);
	}
```
it will result to something like this:
```
Create widget pipeline.0 (pipe 0) - instance 0, core 0
Create widget gain.0.1 (pipe 0) - ID 6, instance 0, core 0
Create widget gain.1.1 (pipe 1) - ID 6, instance 1, core 0
```